### PR TITLE
Styles by hanni

### PR DESCRIPTION
**Pull request by hannnegoossens**
A completely new style for our site!

### CODE REVIEW

File: _components/header.md [hunk 1]
```diff
@@ -21,16 +21,17 @@ This should always be visible, but the `.user` div shouldn't contain all three o
 ## Rendered
 
 <header>
-  <a href="/"><img src="{{site.baseurl}}/src/img/logo.svg" alt="home"></a>
+  <a href="/"><img src="{{site.baseurl}}/src/img/logoBlue.png" alt="home"></a>
   <div class="user">
-    <a href="/me"><img src="{{site.baseurl}}/src/img/user.svg" alt="me"></a>
+    <a href="/me"><img src="{{site.baseurl}}/src/img/userBlue.png" alt="me"></a>
```
Comment (Haroenv) [inline _components/header.md]: Logo in svg

File: _components/poll-input.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This is the input for changing the value of an option of a poll
 ```html
 <div class="poll--option">
   <label for="optionXX">Enter an option</label>
-  <input type="text" name="optionXX">
+  <input type="text" placeholder="Enter an option" name="optionXX">
```
Comment (Haroenv) [inline _components/poll-input.md]: No placeholders

File: _sass/base.scss [hunk 3]
```diff
@@ -7,15 +7,15 @@
 html {
   box-sizing: border-box;
   font-family: $f-sans;
-  color: $c-dark;
+  color: $green;
```
Comment (Haroenv) [inline _sass/base.scss]: No named variables after what it is. Use what it's used for

File: _sass/components/poll.scss [hunk 4]
```diff
@@ -11,10 +11,13 @@
     label {
       font-size: .8em;
       margin-bottom: .5em;
+      display: none;
```
Comment (Haroenv) [inline _sass/components/poll.scss]: Terrible. Labels are visible

File: _sass/variables.scss [hunk 5]
```diff
@@ -9,6 +9,14 @@ $c-accent: #FD6C6C;
 $c-accent-dark: darken($c-accent, 50%);
 $c-dark: #232323;
 
+// colours second version
+$blue: #07092E;
+$green: #4AA07A;
+$green-dark: #367559;
+$green-light: #8BC2AA;
+$red: #B94F4F;
+$red-dark: #5C2828;
```
Comment (Haroenv) [inline _sass/variables.scss]: Don't make new variables. Reuse the old names

**Review by Haroenv — CHANGES_REQUESTED** [review]
Some changes. Overall good idea. Can you show some screenshots in the PR?

File: _components/poll-input.md (hunk 2 above)
Comment (hannnegoossens) [reply _components/poll-input.md]: Why not? Placeholders instead of labels?

Comment (Haroenv) [reply _components/poll-input.md]: Bad user experience. Can't see what you're supposed to fill in without labels

Comment (Haroenv) [reply _components/poll-input.md]: Can give a quote out of heydon's book if you need it

**Comment by hannnegoossens** [comment]
image

image

image

**Comment by ArnaudWeyts** [comment]
maak het minder cancer aub

File: _components/header.md [hunk 6]
```diff
@@ -21,16 +21,17 @@ This should always be visible, but the `.user` div shouldn't contain all three o
...
   <div class="user">
-    <a href="/me"><img src="{{site.baseurl}}/src/img/user.svg" alt="me"></a>
+    <a href="/me"><img src="{{site.baseurl}}/src/img/userBlue.svg" alt="me"></a>
   </div>
 </header>
 
+
 <header>
-  <a href="/"><img src="{{site.baseurl}}/src/img/logo.svg" alt="home"></a>
+  <a href="/"><img src="{{site.baseurl}}/src/img/logoBlue.svg" alt="home"></a>
```
Comment (Haroenv) [inline _components/header.md]: change the fill colour in the svg itself. Don't make a new image 😄

**Comment by Haroenv** [comment]
reference image now:

**Review by Haroenv — APPROVED** [review]

File: _components/header.md [hunk 7]
```diff
@@ -27,10 +27,11 @@ This should always be visible, but the `.user` div shouldn't contain all three o
   </div>
 </header>
 
+
 <header>
   <a href="/"><img src="{{site.baseurl}}/src/img/logo.svg" alt="home"></a>
```
Comment (Haroenv) [inline _components/header.md:32]: Should have a way to log out and show the current user

**Comment by thysruben** [comment]
screenshot 2016-12-11 14 51 27

**Comment by hannnegoossens** [comment]
@ArnaudWeyts, kenkerman, ist nu beter??

**Comment by ArnaudWeyts** [comment]
@hannnegoossens Goed zo 😎

**Pull request merged**